src/ResourceManager/Network/Commands.Network/help/New-AzureRmVirtualWan.md

```diff
@@ -0,0 +1,203 @@
+---
+external help file: Microsoft.Azure.Commands.Network.dll-Help.xml
+Module Name: AzureRM.Network
+online version: https://docs.microsoft.com/en-us/powershell/module/azurerm.network/new-azurermvirtualwan
+schema: 2.0.0
+---
+
+# New-AzureRmVirtualWan
+
+## SYNOPSIS
+Creates an Azure Virtual WAN.
+
+## SYNTAX
+
+```
+New-AzureRmVirtualWan -Name <String> -ResourceGroupName <String> -Location <String>
+ [-AllowVnetToVnetTraffic <Boolean>] [-AllowBranchToBranchTraffic <Boolean>] [-Tag <Hashtable>] [-AsJob]
```

Comment:
All "New" cmdlet should have SwitchParameters instead of booleans.  Apply everywhere.

Reply:
Regenerated help - was already fixed in code.